xtreemfs-common/Dockerfile

```diff
@@ -12,9 +12,11 @@ RUN apt-get -qy install \
     maven \
     openjdk-8-jdk
 
-ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
-ENV JAVA_INCLUDE_PATH /usr/lib/jvm/java-8-openjdk-amd64/include
-ENV JAVA_INCLUDE_PATH2 /usr/lib/jvm/java-8-openjdk-amd64/include/linux
+RUN JAVA_HOME=$(java -XshowSettings:properties -version 2>&1 > /dev/null | grep 'java.home' | grep -oP "/.*")
```

Comment:
I suspect this line does not do anything, as each command is run in another container. Hence, `JAVA_HOME` does not propagate to the next `ENV` instructions. Could you please add `RUN echo ${JAVA_HOME}` just after this line (in your tests, do not commit that), and report the result?

Reply:
I just find it highely unlikely that this should fix the issue. Have you built the images with the `--no-cache` option for `docker-compose`? I suspect that a previous `JAVA_HOME` might still be set. Have you tried `ENV JAVA_HOME /usr/lib/jvm/default-java`? This should be the only change needed in this PR.

Reply:
Just in case I started from a new fresh install on my Raspberry Pi.

The xtreemfs-common building fails with error javac not found! as shown in my first screenshot, visible in the top of the pull request.

I tried with the Dockerfile edited as follow:
![2019-07-04 01_02_24-default-java-Dockerfile](https://user-images.githubusercontent.com/3049704/60629912-0e5df680-9df8-11e9-8f72-5ea261a4944d.png)

Got the same error:
![2019-07-04 01_01_39-default-java-error](https://user-images.githubusercontent.com/3049704/60629921-1c137c00-9df8-11e9-8e1b-8c8f07f3b282.png)
because it seems there is no default-java simlink. I checked before proposing my solution by installing openjdk-8-jdk on the Raspberry itself, and I didn't found anything. That's why I found on internet the proposed solution that was the seemed to be the more elegant and most robust from my point of view.